integration_tests/tests/consistency/consistency_programs.sql

```diff
@@ -0,0 +1,24 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
```

Comment:
Can you add to this model a variable to disable it if the dependent programs are disabled.

Reply:
added

Reply:
added